java/integTest/src/test/java/glide/TestUtilities.java

```diff
@@ -76,4 +84,38 @@ public static Map<String, String> parseInfoResponseToMap(String serverInfo) {
         return RedisClusterClientConfiguration.builder()
                 .address(NodeAddress.builder().port(CLUSTER_PORTS[0]).build());
     }
+
+    @SneakyThrows
+    public static RedisClient createDefaultStandaloneClient() {
+        return RedisClient.CreateClient(commonClientConfig().build()).get();
+    }
+
+    @SneakyThrows
+    public static RedisClusterClient createDefaultClusterClient() {
+        return RedisClusterClient.CreateClient(commonClusterClientConfig().requestTimeout(5000).build())
+                .get();
+    }
+
+    /**
+     * Run a command and expect a response or a request error.
+     *
+     * @param lambda Client command wrapped by a lambda to execute.
+     * @param error An error to expect. Any other error is rethrown.
+     * @return Pair of response + null if command succeeded; or null + error if got the expected
+     *     error.
+     * @param <T> Command return type.
+     */
+    @SneakyThrows
+    public static <T> Pair<T, String> tryCommandWithExpectedError(
```

Comment:
Do we want to ask the AWS team what they think the testing we are doing for save?

Reply:
Sure. @barshaul @shohamazon